ql/src/java/org/apache/hadoop/hive/ql/io/CombineHiveInputFormat.java

```diff
@@ -768,4 +763,8 @@ public String toString() {
   public interface AvoidSplitCombination {
     boolean shouldSkipCombine(Path path, Configuration conf) throws IOException;
   }
+
+  public interface MergeSplit {
```

Comment:
is that generic or only Iceberg specific?

Reply:
is that used at all?

Reply:
This is generic interface for creating merge splits.